### PR TITLE
pr template

### LINKED ISSUE #7: Add github PR template
add a github template with all the checks that an example project must satisfy based on the actual readme project

### PR DESCRIPTION
**Pull request by Eomm**
Closes #7

### CODE REVIEW

**Comment by Eomm** [comment]
PS: I should also ad a template for "example request" (more likes, more we can give it priority)

**Comment by Eomm** [comment]
@mcollina may I improve this?

**Review by mcollina — COMMENTED** [review]
Where example will be located?

I think we should add one case to link new plugins: what needs to be in those repos/what we do to verify them before addition.

**Comment by Eomm** [comment]
> Where example will be located?

In a dedicated folder. This should be already in place in the readme and one bullet point of the PR-template ask if the project is in a standalone dir.

> I think we should add one case to link new plugins: what needs to be in those repos/what we do to verify them before addition.

Clear, maybe this should be a separate PR that enhance the README.md and explain to users the requirements/checks we do?

File: .github/PULL_REQUEST_TEMPLATE/hosted-project.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+the requirements below.
+
+Contributors guide: https://github.com/fastify/example#contributions
+-->
+
+#### Checklist for a new example
+
+- [ ] the project has a dedicated folder in `kebab-case`
+- [ ] there is not an `index.js`, but a valuable name (usaully the same of the folder)
+- [ ] a test example is included (preferring using `tap`)
```
Comment (SerayaEryn) [inline .github/PULL_REQUEST_TEMPLATE/hosted-project.md:17]: I am not sure if we should express a preference for a test framework.

If we do it should be mentioned in the `External example`, too.

Comment (Eomm) [reply .github/PULL_REQUEST_TEMPLATE/hosted-project.md:17]: The idea is that the example in this repo could be updated by us at some point in time, the external no

**Review by SerayaEryn — APPROVED** [review]

**Review by jsumners — APPROVED** [review]

**Review by Ethan-Arrowood — APPROVED** [review]
This looks great! Awesome work @Eomm

**Pull request merged**